settings.py

```diff
@@ -1188,6 +1189,15 @@ def get_user_url(user):
 
 HONEYPOT_FIELD_NAME = 'website'
 
+BLOCKABLE_USER_AGENTS = (
```

Comment:
Please make that setting a list of names and then turn them into a list of regex patterns in the module where the decorator is defined.